core/src/modules/statusReporters.ts

```diff
@@ -528,7 +527,7 @@ function mergeMetrics(metrics: StatusMetric[]) {
       mergedMetrics.push(item);
     } else if (idx !== null) {
       for (const k in Object.keys(item)) {
-        if (item.hasOwnProperty(k)) {
+        if (Object.prototype.hasOwnProperty.call(item, k)) {
```

Comment:
Don't love this rule

Reply:
Looks like [hasOwn](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/hasOwn) exists and we should maybe just use that instead?

Reply:
hasOwn is super new. It looks like we'll have to upgrade to es2022, we're still targeting es2019.

Reply:
I don't object :P